x/staketia/keeper/migration.go

```diff
@@ -253,12 +254,21 @@ func GetStakeibcRedemptionRate(
 	undelegatedBalance := stakeibcKeeper.GetUndelegatedBalance(hostZone.ChainId, depositRecords)
 	nativeDelegation := sdk.NewDecFromInt(hostZone.TotalDelegations)
 
+	// Staketia delegation records are not included in any of the above yet so we must
+	// gather than explicitly
+	// In practice, there will be no records here when the upgrade runs
+	staketiaPendingDelegations := sdk.ZeroDec()
+	for _, delegationRecord := range staketiaKeeper.GetAllActiveDelegationRecords(ctx) {
```

Comment:
just checking: this counts records in `TRANSFER_IN_PROGRESS`, is that the right behavior?

Reply:
Makes sense! I was worried it'd double with the `GetDepositAccountBalance` check which also checks `TRANSFER_IN_PROGRESS` records, but I think that's just for `stakeibc`